release-doc-in-process/all_vf.TTL

```diff
@@ -47,9 +47,13 @@ vf:TransferClassification  a     owl:Class ;
 
 # PLAN CLASSES
 
+vf:Plan     a           owl:Class ;
+        rdfs:label      "vf:Plan" ;
+        rdfs:comment    "A logical collection of processes and/or transfers that constitute a body of planned work with defined deliverable(s)." .
+
 #vf:Intent  a       owl:Class ;
 #        rdfs:label       "vf:Intent" ;
-#        rdfs:comment   "A published want or offer or plan, sometimes with what is expected or desired in return." .
+#        rdfs:comment   "A request or offer or proposal." .
```

Comment:
I think whatever we will come up in #217 we should call `vf:Proposal` or similar but keep `vf:Intent` in line with `vf:Commitment` and `vf:EconomicEvent` - it specifies action and references a resource or classification.

Reply:
I removed this whole definition (it was already commented) so we can include it later without distraction.  I had taken out the word "plan" because it didn't seem correct in that definition.